analyzer/linux/modules/auxiliary/stap.py

```diff
@@ -70,6 +70,9 @@ def has_stap(p):
 
         stap_stop = timeit.default_timer()
         log.info("STAP aux module startup took %.2f seconds", stap_stop - stap_start)
+
+        self.stop()
```

Comment:
is this not called by https://github.com/kevoreilly/CAPEv2/blob/master/analyzer/windows/analyzer.py#L691?

Reply:
oh, yeah - you're right. I definitely overlooked the aux.stop() at line 322; thought it wasn't called because I didn't see it after aux.start() @ 203.